src/main/groovy/wooga/gradle/build/unity/tasks/PlayerBuildEngineUnityTask.groovy

```diff
@@ -0,0 +1,91 @@
+package wooga.gradle.build.unity.tasks
+
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Optional
+
+class PlayerBuildEngineUnityTask extends BuildEngineUnityTask {
+
+    private final Property<String> version
+    private final RegularFileProperty appConfigFile

```

Comment:
I checked the code one more time and I don't see the need for this property. Why can't we just set the `configPath` property directly?

Reply:
Ah nevermind. You want to make the Optional property mandetory in the subclass.